README.md

```diff
@@ -2,55 +2,158 @@
 
 ![Dolphin Guy](http://i.giphy.com/RdkwfjGlsdnRm.gif)
 
-## Learning Objectives - The student should be able to..
+## Objectives
 
-* Explain that the `Bool` is a data type where the value can only be **true** or **false**.
-* Create variables using the Boolean literal values, `true` or `false` like so:
+1. Describe the `Bool` data type
+1. Create Boolean variables with literal values
+1. Use comparison operators on Boolean variables
+1. Use logic operators on Boolean variables
+
+## Understanding Booleans
+
+You are already familiar with creating `Int` variables and constants, which can store single numeric integer values, like
+
+```swift
+var numberOfApples = 7
+let numberOfOranges = 12
+```
+
+In this lesson, we will learn about the boolean data type, which represents a value that can either be true or false. Swift provides constant literal values for both possibilities, which can be used like the integer literals you are already familiar with:
+
+```swift
+let isTheSkyBlue = true
+let canHumansFly = false
+```
+
+Creating and using a variable of this type also works like the types you already know:
+
+```swift
+var isTheGrassGreen = false
+isTheGrassGreen = true
+```
```

Comment:
agree with @JimCampagno here. The reason why that approach works better is that we're being very specific about how this data type and all data types do and shows what happens when you break those roles.